lib/utilities/collectResponseHeaders.js

```diff
@@ -25,6 +25,8 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
     responseMetaHeaders['x-amz-version-id'] =
         getVersionIdResHeader(versioningCfg, objectMD);
 
+    responseMetaHeaders['Accept-Ranges'] = 'bytes';
```

Comment:
Awesome! Thinking out loud: perhaps this could be put near the other standard HTTP headers (e.g., 'Cache-Control', etc.), just below where the Amazon-specific headers (those beginning with 'x-amz-') are set.